frontend/src/components/regions/UserActions/index.jsx

```diff
@@ -6,86 +6,61 @@ import { useParams } from "react-router-dom";
 import { connect } from "react-redux";
 import { apiReduxApps } from "../../../apiAccess/apiApps";
 
-import { ACTION_NAMES } from "./constants";
 import { getLatestMatchingCall } from "@prelude/redux-api-toolbox/src/apiEntities/selectors";
 import Loading from "../../widgets/Loading";
-
-// TODO: look nice
-
-const days = [
-  "Sunday",
-  "Monday",
-  "Tuesday",
-  "Wednesday",
-  "Thursday",
-  "Friday",
-  "Saturday",
-];
+import { eventTypeColors } from "../../../colors";
+import { addEventColorsToLogEntries, matchEventTypesWithColors } from "./utils";
 
 function UserActionsUnconnected({
   authedUserId,
-  projectReviews,
-  cardSummaries,
-  fetchProjectReviewsPages,
   fetchCardCompletions,
   userBurndownStats,
   fetchUserBurndownStats,
+  fetchActivityLogEntries,
+  fetchEventTypes,
+  eventTypes,
+  activityLogEntries,
   // call logs
-  FETCH_RECRUIT_PROJECT_REVIEWS_PAGE,
-  FETCH_USER_ACTIONS_CARDS_COMPLETED_PAGE,
+  FETCH_ACTIVITY_LOG_ENTRIES,
 }) {
   let urlParams = useParams() || {};
   const userId = parseInt(urlParams.userId || authedUserId || 0);
   const currentUserBurndownStats = Object.values(userBurndownStats).filter(
     (snapshot) => snapshot.user === userId
   );
 
-  useEffect(() => {
-    fetchProjectReviewsPages({
-      dataSequence: [
-        { page: 1, reviewerUser: userId },
-        // { page: 1, recruitUsers: [userId] },
-      ],
-    });
-  }, [fetchProjectReviewsPages, userId]);
-
   useEffect(() => {
     fetchCardCompletions({ page: 1, assigneeUserId: userId });
```

Comment:
is this not part of the old activity log aswell?

Reply:
part of burndown